sdk/identity/identity/CMakeLists.txt

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: MIT
+
+cmake_minimum_required (VERSION 3.12)
+
+project (az_identity LANGUAGES C)
+
+set(CMAKE_C_STANDARD 99)
+
+add_library (
+  az_identity
+  src/az_client_secret_credential.c
```

Comment:
**In a subsequent pr** all the identity types should be moved to az_identity_*

Reply:
Thanks! Subsequent PR: https://github.com/Azure/azure-sdk-for-c/pull/275